torch/distributed/tensor/_random.py

```diff
@@ -159,7 +159,7 @@ class OffsetBasedRNGTracker(_RNGStateTracker):
     should be shared and synchronized among all ranks to respect the semantics of DTensor
     random operators.
 
-    note: _RNGStateTracker only supports cuda/cuda-like device
+    note: _RNGStateTracker only supports cuda/cuda-like device. Xpu support is experimental.
```

Comment:
At the current moment, XPU support is WIP and we will enable XPU distributed later as a formal support. It means that the XPU support is not experimental. So, I suggest refining the note a little bit by removing `Xpu support is experimental.` as XPU device is cuda-like device.